.goreleaser.yml

```diff
@@ -50,3 +50,12 @@ release:
     ```
     docker pull ghcr.io/conduitio/conduit:{{ .Tag }}
     ```
+nfpms:
```

Comment:
in general, I don't think we use pre-release and release in our versions, pre-release would look like `v1.0.0~rc1` where rc=release candidate, but yes `prerelease` should be assigned by default
sam e goes for `release`, it would look something like `v0.1.0-1`

Reply:
Let me see if I understood you correctly: if we are releasing Conduit 0.6.0, the resulting Debian package will have the `release` set to `v0.6.0-1` automatically?

Reply:
Technically our nightly releases are prereleases.

Reply:
oh true! but it's still assigned automatically, I just tried releasing a nightly release here https://github.com/ConduitIO/conduit-maha/releases , I downloaded the deb file and used

>  dpkg-deb --show conduit-maha_0.4.0-nightly.20230223_linux_amd64.deb

which shows the pre-release after the `~`, and I got:
`conduit-maha	0.4.0~nightly.20230223`
So it still works.